Freelancer/DATA/EQUIPMENT/weapon_equip.ini

```diff
@@ -51302,3 +51710,70 @@ auto_turret = false
 turn_rate = 90
 LODranges = 0, 100, 200, 500, 1000
 lootable = true
+
+[Munition]
+nickname = santa_gun_plasma_heavy01_ammo
+hp_type = hp_gun
+requires_ammo = false
+hit_pts = 2
+weapon_type = W_Plasma
+hull_damage = 1800
+energy_damage = 0
+one_shot_sound = tachyoncanon
+munition_hit_effect = santa_plasma_01_impact
+const_effect = santa_plasma_01_proj
+lifetime = 1.333333
+force_gun_ori = false
+mass = 1
+volume = 0
+lootable = false
+
+[Gun]
+nickname = santa_gun_plasma_heavy01
+ids_name = 459385
+;res str
+; SANTA’S FROZEN FIRE ($class)
+ids_info = 461005
+;res html
+; \m\bSANTA VON CLAUSSEN’S FROZEN FIRE\B
+; \l
+; Type: $classLong $damageType
+; Hull/Shield Damage: $hullDamage/$energyDamage
+; Range: $range m
+; Projectile Speed: $speed m/s
+; Refire Rate: $refireRate /s
+; Power Draw: $powerUsage
+; Mass: $mass
+; Integrity: $hitpoints
+; \
+; \cYellow\i\bHappy Hollidays\B\I\C

```

Comment:
Holidays with one L and it is missing the ! at the end like the other equips have it.